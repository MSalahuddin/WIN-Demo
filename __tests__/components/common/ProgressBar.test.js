import React from 'react';
import { render } from 'react-native-testing-library';

import PointProgressBar from '../../../js/components/common/ProgressBar';

describe('Component Text', () => {
    test('stack Em progress bar renders correctly ', () => {
        const output = render(<PointProgressBar
            displayProgressInText={false}
            startPoint={0}
            EndPoint={0}
        />);
        expect(output).toMatchSnapshot();
    });
});
