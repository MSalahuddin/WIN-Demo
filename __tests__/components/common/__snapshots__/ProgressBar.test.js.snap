// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Text stack Em progress bar renders correctly  1`] = `
<View
  style={
    Array [
      Object {
        "width": 166,
      },
    ]
  }
>
  <View
    isCenter={true}
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginTop": 0,
        },
      ]
    }
  >
    <View
      bgColor="#F06B59"
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#F06B59",
            "borderRadius": 24,
            "height": 36,
            "justifyContent": "center",
            "width": 36,
          },
        ]
      }
    >
      <Text
        alignCenter={false}
        allowFontScaling={true}
        color="#000000"
        fontFamily="BOLD"
        isUnderlined={false}
        isUppercase={false}
        maxFontSizeMultiplier={1.3}
        size="XXXSMALL"
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "CalibreBold",
              "fontSize": 19.5,
              "justifyContent": "center",
              "textAlign": "left",
              "textDecorationColor": "#000000",
              "textDecorationLine": "none",
              "textDecorationStyle": "solid",
              "textTransform": "none",
            },
            Object {
              "color": "#FFFFFF",
              "lineHeight": 38,
              "marginTop": 4,
            },
          ]
        }
      >
        0
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
            "borderRadius": 12,
            "height": 16,
            "marginHorizontal": 12,
            "width": "70%",
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#FFDB4F",
              "borderRadius": 12,
              "height": 16,
              "position": "absolute",
              "width": "NaN%",
            },
          ]
        }
        width={NaN}
      />
    </View>
    <View
      bgColor="#E5E5E5"
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#E5E5E5",
            "borderRadius": 24,
            "height": 36,
            "justifyContent": "center",
            "width": 36,
          },
        ]
      }
    >
      <Text
        alignCenter={false}
        allowFontScaling={true}
        color="#000000"
        fontFamily="BOLD"
        isUnderlined={false}
        isUppercase={false}
        maxFontSizeMultiplier={1.3}
        size="XXXSMALL"
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "CalibreBold",
              "fontSize": 19.5,
              "justifyContent": "center",
              "textAlign": "left",
              "textDecorationColor": "#000000",
              "textDecorationLine": "none",
              "textDecorationStyle": "solid",
              "textTransform": "none",
            },
            Object {
              "color": "#FFFFFF",
              "lineHeight": 38,
              "marginTop": 4,
            },
          ]
        }
      >
        0
      </Text>
    </View>
  </View>
</View>
`;
